recOrder/plugin/widget/main_widget.py

```diff
@@ -2238,6 +2246,40 @@ def run_calibration(self):
 
         self.worker.start()
 
+    def _check_MM_config_setup(self):
```

Comment:
This function isn't particularly beautiful, but none of our mmc code is and it's just for warnings.